modules/xmpp/JingleSessionPC.js

```diff
@@ -1007,12 +1007,27 @@ export default class JingleSessionPC extends JingleSession {
                 // modify sendSessionAccept method to do that
                 this.sendSessionAccept(() => {
                     success();
-
                     this.room.eventEmitter.emit(XMPPEvents.SESSION_ACCEPT, this);
+
+                    // The first video track is added to the peerconnection and signaled as part of the session-accept.
+                    // Add secondary video tracks (that were already added to conference) to the peerconnection here.
+                    // This will happen when someone shares a secondary source to a two people call, the other user
+                    // leaves and joins the call again, a new peerconnection is created for p2p connection. At this
+                    // point, there are 2 video tracks which need to be signaled to the remote peer.
+                    const videoTracks = localTracks.filter(track => track.getType() === MediaType.VIDEO);
+
+                    videoTracks.length && videoTracks.splice(0, 1);
+                    if (FeatureFlags.isMultiStreamSupportEnabled() && videoTracks.length) {
+                        // Clear the conference field on the secondary local tracks if it was set previously, this is
+                        // needed for correctly identifying the RTCRtpSender for track replacement.
+                        videoTracks.forEach(track => {
```

Comment:
is this needed only for the p2p case?

Reply:
This would be needed for jvb case as well. A user shares a secondary video stream, then everyone else leaves the call and all the peerconnections on the ep are closed after the idle timeout. When someone new joins the call, then new p2p and jvb connections are created. The conference field needs to be set to null for all the secondary tracks.